haiku-apps/bephonebook/bephonebook-1.2.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="A phone book manager"
+DESCRIPTION="BePhoneBook is a colorful phonebook made to help you stay \
+organized with your family's or friends' phone numbers."
+HOMEPAGE="https://github.com/HaikuArchives/BePhoneBook"
+COPYRIGHT="1999 Michael Melloff \
+	Fabien Fulhaber"
+LICENSE="BePhoneBook"
+REVISION="1"
+srcGitRev="69d6a5ada4fa76b0e900face8f6eef1d7ac48119"
+SOURCE_URI="https://github.com/HaikuArchives/bephonebook/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="31a4b3c72905a789746509801ee667ebb1ef3b7dbd57c63623b83fc4b064ccc6"
+SOURCE_FILENAME="BePhoneBook-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="BePhoneBook-$srcGitRev"
+PATCHES="bephonebook-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 x86_64"
```

Comment:
```diff
-ARCHITECTURES="?x86_gcc2 x86_64"
+ARCHITECTURES="x86_gcc2 x86_64"
```
(The app builds fine with gcc2 and seems to also work the same way it does on x86_64.)